### PR TITLE
Add support for JHEMCU GF16 F405 - JHEF405

### PR DESCRIPTION
**Pull request by rickard-von-essen**
This adds support for JHEMCU GF16 F405 flight controller[1]. The config was based on the 4.3.0 cli dump provided in the manufacturers Github repo[2], `GF16-F405-BMI.txt` in the `FC.rar` archive.

[1] https://www.jhemcu.com/e_productshow/?76-JHEMCU-GF16-BMI-stack-F405-Flight-control-13A-ESC-76.html
[2] https://github.com/jhemcu/FC-ESC-Firmware/tree/main/GF16-BMI%2B13A%20ESC

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Comment by nerdCopter** [comment]
- needs sync to https://github.com/betaflight/unified-targets
- thank you, typically the manufacturer should be doing this. i'm not sure the protocol for acceptance.

**Comment by ot0tot** [comment]
Isn't this the same as the existing `JHEF405PRO` target?
https://github.com/betaflight/config/blob/master/configs/JHEF405PRO/config.h

File: configs/JHEF405/config.h [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    TIMER_PIN_MAP( 4, PB5 , 1, 0) \
+    TIMER_PIN_MAP( 5, PB7 , 1, 0) \
+    TIMER_PIN_MAP( 6, PC9 , 2, 0) \
+    TIMER_PIN_MAP( 7, PC8 , 2, 0) \
+    TIMER_PIN_MAP( 8, PA9 , 1, 0)
+
+#define ADC1_DMA_OPT        0
+
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_SERIAL
```
[the remark this answers is not in the document]
Comment (rickard-von-essen) [reply configs/JHEF405/config.h:99]: Hm, let me double check this. I checked that blackbox and OSD works.

**Comment by haslinghuis** [comment]
@rickard-von-essen Did not verify or compare yet but as @ot0tot points out this target is already available as JHEF405PRO.

**Comment by rickard-von-essen** [comment]
> @rickard-von-essen Did not verify or compare yet but as @ot0tot points out this target is already available as JHEF405PRO.

Pretty sure the JHEF405PRO is this AIO:
https://www.jhemcu.com/e_productshow/?78-GHF405AIO-ICM-78.html

**Comment by ot0tot** [comment]
> > @rickard-von-essen Did not verify or compare yet but as @ot0tot points out this target is already available as JHEF405PRO.
> 
> Pretty sure the JHEF405PRO is this AIO: https://www.jhemcu.com/e_productshow/?78-GHF405AIO-ICM-78.html
> 
> (and that I tried that ~ 4 months ago when I started working on this.. IIRC)

They use the same target. Do a diff, you'll see they are the same.

**Pull request closed without merging**